scripts/macOS-decrypt.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# This script decrypts our test database
+# Bash3 Boilerplate. Copyright (c) 2014, kvz.io
+# You will need the environment variables CIRCLE_CI_KEY and CIRCLE_CI_IV
+# Run from /dockstore. "bash scripts/macOS-decrypt.sh"
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+
+docker run -it --volume $(pwd):/home/project --env CIRCLE_CI_KEY --env CIRCLE_CI_IV  ubuntu:latest bin/bash home/project/scripts/decrypt-in-ubuntu-container.sh
```

Comment:
May want to lock the Ubuntu version instead of `latest`, although it's unlikely something like this would break based on the version.

Reply:
Alternatively, could test this (optional) but it seems unlikely to be useful since we're the only ones that would use this
https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners
https://circleci.com/docs/2.0/testing-ios/#using-the-macos-executor